### PR TITLE
remove deviceadm

### PR DESCRIPTION
**Pull request by mchalski**
https://tracker.mender.io/browse/MEN-1956

### CODE REVIEW

File: 02.Architecture/02.Device-authentication/docs.md [hunk 1]
```diff
@@ -49,77 +44,68 @@ important to track those, and allow the user to admit or reject a particular ide
...
 | ![Preauthorization flow](preauth.png) |
 |:--:|
 |*Preauthorization flow*|
 
-! It is important to note that the Device Authentication API used here, although exposed as a *management API*, is not meant to be called directly by the user at any time. It is strictly internal and will be hidden in upcoming releases of Mender.
-
-For details of API calls please consult the API documentation:
-
-* [Device Admission Management API](../../apis/management-apis/device-admission)
+* [Device Admission Management API](../../apis/management-apis/device-admission) TODO fix link
```
Comment (drewmoseley) [inline 02.Architecture/02.Device-authentication/docs.md]: Should we resolve this TODO before merging?

File: 06.Server-integration/02.Preauthorizing-devices/docs.md [hunk 2]
```diff
@@ -124,7 +103,7 @@ Secondly, set the contents of the device public key you generated above in a sec
 DEVICE_PUBLIC_KEY="$(cat keys-client-generated/public.key | sed -e :a  -e 'N;s/\n/\\n/;ta')\n"
 ```
 
-Then simply call the [API to preauthorize a device](../../apis/management-apis/device-admission#devices-post):
+TODO will be rendered diff. Then simply call the [API to preauthorize a device](../../apis/management-apis/device-admission#devices-post):
```
Comment (drewmoseley) [inline 06.Server-integration/02.Preauthorizing-devices/docs.md]: Should this TODO be resolved?

File: 06.Server-integration/02.Preauthorizing-devices/docs.md [hunk 3]
```diff
@@ -84,29 +84,8 @@ Now that we have the device's identity and public key, we will use the Mender se
 First make sure to power off your device, so it does not continuously appear as pending in your server.
 
 We recommend that you ensure there are no records of your device in the server; open the Mender UI, then go to *Devices* to see if it is there, then *Decommission* it.
-Secondly, To make sure that the device is not in an existing authentication set, we check *both* the `admission` and `devauth` services for the identity of your device.
```
Comment (estenberg) [inline 06.Server-integration/02.Preauthorizing-devices/docs.md]: Is this not needed anymore since the new service will be more stable?

**Comment by estenberg** [comment]
Looks good to me, but should remove the TODOs before merging like @drewmoseley mentions; I assume this is already the plan.

**Comment by mchalski** [comment]
@drewmoseley @estenberg both TODOs removed, thanks

**Comment by drewmoseley** [comment]
@maciejmrowiec @kjaskiewiczz any thoughts before this is merged?

**Pull request merged**